### ISSUE TITLE
[Feature Request] Color size text by improvement or not

### ISSUE THREAD

**Issue by Stanzilla** [xanderfrangos/crushee#7]
It would be cool if the "x% smaller/ larger" text was colored depending on whether it's an improvement or not, so red for larger and green for smaller.

**Comment by xanderfrangos** [comment]
I agree. That's one of a few UI tweaks I intend to make to the files list. It's a little hard to digest what changed with an image at a glance. A few things I have in mind are:

- Make the % change more obvious (w/ color and a small arrow to indicate larger or smaller).
- Make the file name smaller (it's not that important).
- Show image dimensions (original and new).
- Make it clearer what the original and new file sizes are.
